shared/src/akri/configuration.rs

```diff
@@ -258,23 +278,25 @@ mod crd_serialization_tests {
                     ],
                     "type": "ClusterIP"
                 },
-                "brokerPodSpec": {
-                    "containers": [
-                        {
-                            "image": "nginx:latest",
-                            "name": "usb-camera-broker",
-                            "resources": {
-                                "limits": {
-                                    "{{PLACEHOLDER}}": "1"
+                "brokerSpec": {
```

Comment:
should there be some tests with brokerJobSpec?

Reply:
Sure i can add one, but what would this check beyond verifying that serde works properly?

Reply:
i would phrase it differently ... it checks that we've asked serde to do what we think we did

Reply:
okay can add!

Reply:
added in 6e1e8817ba6a34b99f3